openwisp_monitoring/device/static/monitoring/css/daterangepicker.css

```
@@ -138,9 +138,9 @@
   cursor: pointer;
 }
 .daterangepicker .calendar-table {
-  border: 1px solid #fff;
+  border: 1px solidvar(--ow-color-white);
```

Biome (2.4.7)

[error] Unexpected unknown function: solidvar

(lint/correctness/noUnknownFunction)